src/components/githubLandingPage/AccuracySection/AccuracySection.tsx

```diff
@@ -0,0 +1,296 @@
+import React, { useState } from 'react'
+import Section from '../../common/Section'
+import Container from '../../common/Container'
+
+import { ReactComponent as LineSVG } from './LineSVG.svg'
+import classNames from 'classnames'
+
+import styles from './AccuracySection.module.scss'
+import { AnimatePresence, motion } from 'framer-motion'
+
+enum xOptions {
+  X1 = '0',
+  X2 = '1',
+  X3 = '2',
+  X4 = '3',
+}
+
+enum tabOptions {
+  Accuracy,
+  Stability,
+}
+export default function AccuracySection() {
+  const chartWidth = 860
+  const chartHeight = 414
+  const xPoints = 4
+  const maxY = 100000
+  const offset = 25
+  const yPointsPro = [99500, 99003, 98507, 98015]
+  const xPointsLine = ['30 days', '60 days', '90 days', '120 days']
+  const yPointsLine = ['100k', '75k', '50k', '25k', '0']
```

Comment:
Minor: d3 calls this a scale:

```suggestion
  const xScale = ['30 days', '60 days', '90 days', '120 days']
  const yScale = ['100k', '75k', '50k', '25k', '0']
```